tests/unit/state.py

```diff
@@ -0,0 +1,11 @@
+from dvc.state import State
+from tests.basic_env import TestDvc
+
+
+class TestMtimeAndSizeReturnType(TestDvc):
+    def test(self):
+        self.create(self.DATA, self.DATA_CONTENTS)
```

Comment:
Already created https://github.com/iterative/dvc/blob/0.23.2/tests/basic_env.py#L59 :slightly_smiling_face: 
```suggestion
        self.create(self.DATA, self.DATA_CONTENTS)
```